pom.xml

```diff
@@ -1280,6 +1280,24 @@
                     <artifactId>jacoco-maven-plugin</artifactId>
                     <version>0.8.8</version>
                 </plugin>
+                <plugin>
+                    <groupId>org.kordamp.maven</groupId>
+                    <artifactId>pomchecker-maven-plugin</artifactId>
+                    <version>1.8.0-SNAPSHOT</version>
```

Comment:
I upgraded to this SNAPSHOT to test kordamp/pomchecker#9; works nicely! We can merge once 1.8.0 is released :)